src/planet-4-151612/wordpress/wp-config.php.tmpl

```diff
@@ -61,6 +61,7 @@ define('NONCE_SALT',       '{{ .Env.WP_NONCE_SALT }}');
 define( 'WP_DEBUG', true );
 define( 'WP_DEBUG_LOG', true );
 define( 'SCRIPT_DEBUG', true );
+define( 'ALLOW_THEME_EDITOR_NON_LOGGED_IN', true );
```

Comment:
Do you think it would make sense to have that as a more generic and futureproof name?
eg. `ALLOW_EXPERIMENTAL_FEATURES` or something like that.

Reply:
Absolutely, that would make it easier to experiment :100: